### ISSUE TITLE
Migrations don't work with IDEs

### ISSUE THREAD

**Issue by TrevorKarjanis** [angular-eslint/angular-eslint#617]
IDEs like VSCode lint the project with TSLint and the root application with ESLint after migrating only a project library. I have created a [repository that reproduces the issue](https://github.com/TrevorKarjanis/angular-eslint-issue-617).
1. Create an Angular 11 workspace with a root application and library project.
2. (Optional) Upgrade to Angular 12.
3. [Migrate the library](https://github.com/angular-eslint/angular-eslint#step-2---run-the-convert-tslint-to-eslint-schematic-on-a-project) project to use @angular-eslint.
  ```bash
  > ng add @angular-eslint/schematics
  > ng g @angular-eslint/schematics:convert-tslint-to-eslint test-lib
  ```
4. Fix any linter errors, and `npm run lint` should pass.
5. Open the workspaces in VSCode, and enable TSLint and ESLint.
6. Notice `projects/test-lib/src/lib/test-lib.component.ts` reports that `lib-test-lib` has an invalid prefix.
7. Change the root ESLint configuration, e.g. `"indent": ["error", 4]`.
8. Notice `src/app/app.module.ts` is linted with ESLint.

I had expected the migration to handle this, but it appears I am able to work around it by making the following modifications.
Is this correct, expected, or are there alternative recommendations?
```json
.eslintrc.json

"ignorePatterns": [
  "projects/**/*",
  "src/**/*"
]
```
```json
tslint.json

"linterOptions": {
  "exclude": ["projects/test-lib/**/*"]
}
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/builder`          | `12.3.1` |
| `ESLint`                           | `7.26.0` |
| `node`                             | `12.18.1` |
| `VSCode`                             | `1.59.0-insider (Universal)` |

```sh
Angular CLI: 12.1.4
Node: 12.18.1
Package Manager: npm 6.14.5
OS: darwin x64

Angular: 12.1.4
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1201.4
@angular-devkit/build-angular   12.1.4
@angular-devkit/core            12.1.4
@angular-devkit/schematics      12.1.4
@schematics/angular             12.1.4
ng-packagr                      12.1.2
rxjs                            6.6.7
typescript                      4.3.5
```

- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by JamesHenry** [comment]
Hi @TrevorKarjanis, I'm really sorry that you didn't receive an explicit response on this one. I am going to close it due to its age and lack of general activity of the last few years. I am inferring from that that it wasn't a widespread issue with users, and at this point the tooling has moved on from migrating people from TSLint. Many thanks!
